hybris/egl/glvnd/generate/eglFunctionList.py

```diff
@@ -0,0 +1,238 @@
+"""
```

Comment:
Missing license header is this also under MIT like other added .py files?

Reply:
Probably yes; the file originally comes from Mesa. The original file doesn't have the notice either, but I think we can assume the same license as the rest of Mesa.
https://github.com/mesa3d/mesa/blob/main/src/egl/generate/eglFunctionList.py

Although, I would say that this file contains mostly data and not much code, and so might be not copyrightable. In any case, I'll update another version with the header some time soon.